suite-native/device-manager/src/components/DeviceList.tsx

```diff
@@ -93,25 +94,21 @@ export const DeviceList = ({ isVisible, onSelectDevice }: DeviceListProps) => {
         if (isVisible) {
             const hasNotSelectedInstancelessDevices = notSelectedInstancelessDevices.length > 0;
```

Comment:
You are defining just the same variable few lines under this useEffect. Remove it here, and define it only once.

Reply:
f312d14b01